src/modules/GameConstants.ts

```diff
@@ -1627,18 +1628,18 @@ export const HisuiDungeons = [
     'Ancient Lake Valor',
     'Ancient Lake Acuity',
     'Temple of Sinnoh',
-    'Turnback Cave', // 211
+    'Turnback Cave', // 212
 ];
 
 export const PaldeaDungeons = [
-    'Inlet Grotto', // 211
+    'Inlet Grotto', // 212
```

Comment:
There was a misalignement. It should be 213.
Can you take advantage of the PR to fix this index and that of Area Zero Depths (220), please ?